src/program/snabbvmx/tests/selftest.sh

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/env bash
```

Comment:
I'm not a fan of bash either but it's what Snabb uses for tests, so not really a choice here.